astroplan/core.py

```diff
@@ -295,15 +354,14 @@ def parallactic_angle(self, time, target):
         if not isinstance(time, Time):
             time = Time(time)
 
-        if not (hasattr(target, 'ra') or hasattr(target, 'dec') or
-                hasattr(target, 'coord')):
-            raise TypeError('The target must be a coordinate (i.e. a '
-                            'FixedTarget or SkyCoord).')
-
```

Comment:
If we really must have a `SkyCoord` here (or `Target` with a SkyCoord), then I think an `isinstance` test would be better. i.e.
```python
if not isinstance(target, SkyCoord) or isinstance(target, Target):
     ....
```
Otherwise I understand this a good test for duck-typed objects.